src/components/MultiHopTrade/hooks/useGetTradeQuotes/useGetTradeQuotes.tsx

```diff
@@ -103,7 +103,7 @@ export const useGetTradeQuotes = () => {
   const sellAccountId = useAppSelector(selectSellAccountId)
   const buyAccountId = useAppSelector(selectBuyAccountId)
 
-  const slippageTolerancePercentage = useAppSelector(selectTradeSlippagePercentageDecimal)
```

Comment:
This selector uses the default slippage of the _currently selected_ swapper, which makes no sense when fetching quotes, as each swapper has different defaults and so can't share the same input args (unless they've been overwritten by the user to a specific value, as is achieved by `selectSlippagePreferencePercentageDecimal`).